pgmpy/continuous/discretize/unbiased.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import division
+
+import numpy as np
+from scipy import integrate
+
+from pgmpy.factors import ContinuousFactor
+from pgmpy.continuous.discretize import BaseDiscretizer
+
+
+class UnbiasedDiscretizer(BaseDiscretizer):
+    """
+    This class uses the unbiased method for discretizing the
+    given continuous distribution.
+
+    The unbiased method for discretization is the matching of the
+    first moment method. It involves calculating the first order
+    limited moment of the distribution which is done by the _lim_moment
+    method.
+
+    For this method,
+
+    The probability mass in x=[low] is,
+    (E(x) - E(x + step))/step + 1 - cdf(x)
+
+    The probability mass in x=[low+step, low+2*step, ........., high-step],
+    (2 * E(x) - E(x - step) - E(x + step))/step
+
+    for x=[high],
+    (E(x) - E(x - step))/step - 1 + cdf(x)
+
+    where, E(x) is the first limiting moment of the distribution
+    about the point x, cdf is the cumulative density function
+    and step = (high-low)/cardinality.
+
+    For details, refer Klugman, S. A., Panjer, H. H. and Willmot,
```

Comment:
@yashu-seth Add a separate section for references (like parameters, examples). Search for reference in pgmpy.